sphinxext/photofinish/__init__.py

```diff
@@ -142,6 +148,13 @@ def visit_image(
                     im_width, im_height = im.size
                 soup_img.attrs["height"] = im_height
                 soup_img.attrs["width"] = im_width
+            except NoToolError as e:
+                if not SVG_NO_TOOL_MSG_PRINTED:
+                    print(e)
+                    logger.info(
+                        "No conversion tool was found. Please install one of: cairosvg, svglib, inkscape, rsvg-convert, svgexport, or imagemagick. If you have one of these installed, they may not be on your path. Pass your installed tool's path into this function."
+                    )
```

Comment:
A: I think the actual error message's contents should remain in svg2png in case we wan't to reuse the function.
```suggestion

```